BackendServices/AuxiliaryServices/WebAPIService/WebAPIService.csproj

```diff
@@ -14,6 +14,7 @@
   
   <ItemGroup>
     <ProjectReference Include="..\..\NetworkLibrary\NetworkLibrary.csproj" />
+    <ProjectReference Include="..\..\WatsonWebserver\WatsonWebserver.csproj" />
```

Comment:
Is it necessary? What method do you use from Watson? If you need one I can move it to a more general purpose library.

Reply:
Without notice sometime Visual Studio add these 😓 I removed